pages/components/SubmitButtonComponent.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+interface SubmitButtonComponentProps {
+  responses: { [key: string]: string };
+}
+
+const SubmitButtonComponent: React.FC<SubmitButtonComponentProps> = ({
+  responses,
+}) => {
+  const handleSubmit = () => {
+    console.log('Submitted Responses:', responses);
+    
+  };
+
+  return (
+    <button 
+      onClick={handleSubmit} 
+      className="bg-blue-500 text-white p-2 rounded"
```

Comment:
Looks good; I would recommend adding some more styling like a hover effect where the opacity changes on the mouse hover and some more padding for the width